src/test/java/com/exasol/adapter/dynamodb/DynamodbTestUtils.java

```diff
@@ -119,9 +124,13 @@ public void pushBook(final String isbn, final String name) {
 	 * @return number of scanned items
 	 */
 	public int scan(final String tableName) {
-		final ScanResponse result = this.dynamoClient.scan(ScanRequest.builder().tableName(tableName).build());
-		LOGGER.trace(result.toString());
-		return result.count();
+		final Table table = this.dynamoClient.getTable(tableName);
+		final ItemCollection<ScanOutcome> scanResult = table.scan();
+		for (final Item item : scanResult) {// it is important to scan all items here for getAccumulatedScannedCount to
```

Comment:
Extract method with speaking name like `scanAllItemsBeforeCount()` and remove comment.